### DISCUSSION TITLE
Having trouble transitioning from gameplay to Dialogic timeline

### DISCUSSION THREAD

**Discussion by VoltitanDev** [dialogic-godot/dialogic#2254] [Q&A]
I'm currently developing a visual novel/3rd person mecha hybrid game while I have been able to get the dialogic segments to transition from VN to gameplay I was wondering how to accomplish the opposite from gameplay to the visual novel segment I've tried using the gamemanager(custom made to transition from 3d to 2d that works well for Dialogic 1 but currently is struggling with Dialogic 2) for when the player wins but so far nothing happens. However, If when using the Dialogic.start(insert_timeline) directly in the level's script the dialogue itself appears but not the backgrounds

level script:
![image](https://github.com/dialogic-godot/dialogic/assets/162152507/6f3b8a68-ff0c-467e-9d24-c4431903d99a)

Game manager:
![image](https://github.com/dialogic-godot/dialogic/assets/162152507/692206c1-fdb8-4f12-bc6a-66ae6748ddda)

Gameplay screenshot
![image](https://github.com/dialogic-godot/dialogic/assets/162152507/bcd9ff20-5809-40dc-9106-17190cb62a96)

Edit: Turns out its mostly a dependency issue for the background image but I'm currently having issues changing to the dialogic timeline segment fully when the level clears since I want it to move to the VN segment as if it were changing scenes rather than the vn overlaying over the gameplay(The 3d character's inputs appear to be interfering with the vn inputs during the timeline segment).

**Comment by NicholasDewberryOfficial** [comment]
Just saw this, Do you still need help with this?

Since you're trying to switch the scene completely to run the VN on its own (instead of having it overlay), you'll have to switch it to a new scene.

Create a new scene with nothing on it, and in that scenes func _ready() script load the dialogic timeline. Then, instead of loading the dialogic script in the gameplay scene, change the scene to the VN one, then you should be good. 

Does that make sense?

**Reply by VoltitanDev** [reply]
I resolved that issue a few months ago but I appreciate the answer.
